openedx/core/djangoapps/password_policy/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not a comment for this WIP PR, but a note for when this is ready to land:

We may need/want to separate the migrations from the application code and ship them first.  We may also need to consider how we anticipate the rollback code to be used.  If it just deleted the group and we don't expect there to be any/many users this is likely fine.  But if we anticipate this getting called anytime after users are being put into this group this will also cleanup the per-user join records which could take a while depending on the size.